ui/Modal/Org/ConfigureEns/BlockTimer.svelte

```diff
@@ -0,0 +1,49 @@
+<!--
+ Copyright © 2021 The Radicle Upstream Contributors
+
+ This file is part of radicle-upstream, distributed under the GPLv3
+ with Radicle Linking Exception. For full terms see the included
+ LICENSE file.
+-->
+<script lang="typescript">
+  import { onDestroy } from "svelte";
+
+  import * as svelteStore from "ui/src/svelteStore";
+  import * as wallet from "ui/src/wallet";
+
+  export let startBlock: number;
+  export let minimumCommitmentAge: number;
+  export let onFinish: () => void;
+
+  const walletStore = svelteStore.get(wallet.store);
+
+  const requiredBlockCount = minimumCommitmentAge + 1;
+  let confirmedBlockCount: number = 0;
+  let done: boolean = false;
```

Comment:
We changed the flow, now this screen contains also the "Confirm registration" button which gets enabled once the necessary block count is reached. This boolean prevents the block count from continuing to increase after the required amount is reached if the user doesn't click the button right away.

If we don't do that it will say "Confirmed 12 of 11 blocks" after a while.

<img width="841" alt="Screenshot 2021-08-30 at 11 45 07" src="https://user-images.githubusercontent.com/158411/131320626-65db130f-70ae-4e0d-8d35-28308f6b34e3.png">

Reply:
> If we don't do that it will say "Confirmed 12 of 11 blocks" after a while.

Does it? We remove the block listener after calling `onFinish()` so we should stop updating the block nubmer.

Reply:
Yes, it does. `onFinish()` doesn't advance to the next screen anymore and doesn't thereby trigger `onDestroy`.

Reply:
🤦 never mind.
856bbb93e46fd913704d97861efbce4497354e0c